src/diario/login-aluno.html

```diff
@@ -22,9 +22,9 @@ <h5 class="margin-0">Diário</h5>
 			<div class="col s12 m4 l4 full-heigth" style="margin-top:10rem;">
 				<div class="center-align">
 					<h6 class="primary white-text " style="height: 3rem;line-height: 3rem;">LOGIN</h6>
-					<input type="text" placeholder="CPF" style="margin-top: 5rem;"><br>
-                    <input type="password" placeholder="Senha" style="margin-bottom: 5rem;"><br>
-                    <a class="primary alt waves-effect waves-light btn-large col s12"
+					<input id="cpf" type="text" placeholder="CPF" style="margin-top: 5rem;" maxlength="11"><br>
```

Comment:
* Se o campo de CPF aceita até 11 caracteres, significa que ele deve aceitar apenas números. Então, o ideal, nesse caso, seria deixar o tipo do campo como number.
* Outra opção: deixar o tipo do campo como texto, porém, aceitar também o formato 000.000.000-00.
```suggestion
					<input id="cpf" type="number" placeholder="CPF" style="margin-top: 5rem;" maxlength="11"><br>
```